### DISCUSSION TITLE
[BUG] App crashes

### DISCUSSION THREAD

**Discussion by MidlajYH** [community/community#43446] [Mobile] [answered]
https://user-images.githubusercontent.com/85442225/210938605-b1f4731c-ba5a-459b-8104-a4e7e94b4d9c.mp4

Device: Poco X3 Pro
Android Version: 12

Using Github Mobile Beta v1.95.0-beta (10104)

**Comment by stevepopovich** [comment, marked as answer]
Hi! We are aware of this issue and will have fix coming for this in the next release. Thanks for reporting!
